docs/developer/attributes.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Attributes
+sidebar_label: Attributes
+---
+
+## Introduction
+
+Setting up attributes will allow your users to filter your products catalogue.
+
+Attributes can be used for
+- [Products](developer/api-reference/objects/product.mdx) and [ProductVariants](developer/api-reference/objects/product-variant.mdx)
+- [Pages](developer/api-reference/objects/page.mdx)
+
+## Use cases
+
+1. Your store sells Mugs in many colors and sizes. This means you will need:
+- Color attribute (red, black, etc.)
+- Size attribute (Small, Big)
+
+2. Your shop has many Pages which should render with different styles. To achieve that, you can create attribute named `Template` with type `PAGE_TYPE`. As values you could set: `Support`, `Blog`, `Promos`. Then in storefront code you could load proper css template based on page `Template` value.
```

Comment:
Your shop has many pages with different rendering styles. You can create an attribute called `Template` with a `PAGE_TYPE` type. You can set the following values: `Support`, `Blog`, `Promos`. Then you can upload a proper CSS template based on the `Template` value in the storefront code.